python/tester/utils.py

```diff
@@ -220,7 +220,7 @@ def send_next_visit(url, group, visit_infos):
                    f"filters: {info.filters} ra: {info.position[0]} dec: {info.position[1]} "
                    f"instrument: {info.instrument} survey: {info.survey}")
         records_level = dict(value=asdict(info))
-        value_schema_level = dict(value_schema_id=1, records=[records_level])
+        value_schema_level = dict(value_schema_id=97, records=[records_level])
```

Comment:
I'm a bit worried about this arbitrary-looking number. Will we have to update this every time something changes in the `dev` Kafka?

Reply:
It seems yes. I'm also worried about the not-very-well-documented update process related to Sasquatch redeployment and schema registry changes.

Reply:
We might alternatively send the messages with the full schema every time (a long string with all fields, their types. The `value_schema` on [Sasquatch doc](https://sasquatch.lsst.io/user-guide/restproxy.html)).  I'm not sure that'd be better in this case.

Reply:
Probably not, if that's not what the summit is doing.

Reply:
I couldn't track down what summit does and will just continue to use schema id for now. I'll add some more docs in [DM-42618](https://jira.lsstcorp.org/browse/DM-42618)